src/js/pensions/pensions-entry.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createHistory } from 'history';
+import { Router, useRouterHistory } from 'react-router';
+import { Provider } from 'react-redux';
+
+import initReact from '../common/init-react';
+import route from './routes';
+import createCommonStore from '../common/store';
+import createLoginWidget from '../login/login-entry';
+import reducer from './reducer';
+
+require('../common');
```

Comment:
Ohh, interesting. `require()`ing to inject all the js from `../common/index.js`, aye? I was a bit befuddled why we were mixing `import` and `require()`, but I think I get it.

I imagine we'd only ever do this at the entry point, yes?